pkgs/development/tools/misc/ttags/default.nix

```
@@ -0,0 +1,38 @@
+{ lib , fetchFromGitHub , rustPlatform }:
+
+rustPlatform.buildRustPackage {
+  pname = "ttags";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "npezza93";
+    repo = "ttags";
+    rev = "b8ced4cc71bbf27f514f0084b58e1808a0ea4164";
+    hash = "sha256-nWtsON6VEgXlBao7jxUMIvGJ/8k6cw1I/Zd+aSYzV1I=";
+  };
+
+
+  cargoHash = "sha256-JI7hVmIQRkWZR3ftHFVPOWDTMZB+lfq5cKXCJOZC4Og=";
+
+  meta = with lib; {
+    description = "Generate tags using tree-sitter";
+    longDescription = ''
+      ttags generates tags (similar to ctags) for various
+      languages, using tree-sitter.
+
+      Can be run as a language server that updates the tags
+      for a file when it is saved.
+
+      Supported languages:
+      - JavaScript
+      - Ruby
+      - Rust
+      - Haskell
+    '';
+    homepage = "https://github.com/npezze03/ttags";
```

```suggestion
    homepage = "https://github.com/npezza93/ttags";
```
there is a minor typo in this link

---

Good catch!